public/data/javascript.json

```diff
@@ -798,6 +798,20 @@
   ],
   "tags": ["color", "hex", "contrast", "brightness", "utility"],
   "author": "yaya12085"
+},
+{
+  "title": "Random string",
+  "description": "Generates a random string of characters of a certain length",
+  "code": [
+    "function makeid(length) {",
+    "  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';",
```

Comment:
Maybe you could pass the characters as an argument, making the current value you have here the default.